src/util.js

```diff
@@ -134,3 +134,92 @@ export function getCommitArr(arr, cmd) {
     }
     return ret;
 }
+
+/*
+ * The functions quote_cmdline and unquote_cmdline implement
+ * a simple shell-like quoting syntax.  They are used when letting the
+ * user edit a sequence of words as a single string.
+ *
+ * When parsing, words are separated by whitespace.  Single and double
+ * quotes can be used to protect a sequence of characters that
+ * contains whitespace or the other quote character.  A backslash can
+ * be used to protect any character.  Quotes can appear in the middle
+ * of a word.
+ */
+
+export function quote_cmdline(words) {
```

Comment:
Do these come from Cockpit? Please mention that in the comment above so that people know where to apply fixes and pull in new versions from.

Reply:
Done